app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -121,6 +121,36 @@ def alternate_content_sources
         SmartProxy.joins(:smart_proxy_alternate_content_sources).where('katello_smart_proxy_alternate_content_sources.smart_proxy_id' => self.id)
       end
 
+      def update_content_counts!
```

Comment:
Wondering if this should be a separate task which is called as an async task at end of capsule sync and the update_counts API can also trigger this as a task since there are all the capsule calls involved. We got capsules on ⛵ to tend to.. 😸

Reply:
Sure I can do that :+1:

Reply:
Actually, this will be done at the end of the smart proxy sync since it's being run as part of `SyncCapsule`. It's run right before the container gateway data sync.